website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -48,17 +48,21 @@ Interacting with notification configurations requires admin access to the releva
 
-| Display Name    | Value                   | Description                                                                                              |
-| --------------- | ----------------------- | -------------------------------------------------------------------------------------------------------- |
-| Created         | `"run:created"`         | When a run is created and enters the "Pending" state.                                                    |
-| Planning        | `"run:planning"`        | When a run acquires the lock and starts to execute.                                                      |
-| Needs Attention | `"run:needs_attention"` | Human decision required. When a plan has changes and is not auto-applied, or requires a policy override. |
-| Applying        | `"run:applying"`        | When a run begins the apply stage, after a plan is confirmed or auto-applied.                            |
-| Completed       | `"run:completed"`       | When the run has completed on a happy path and can't go any further.                                     |
-| Errored         | `"run:errored"`         | When the run has terminated early due to error or cancellation.                                          |
+| Display Name       | Value                   | Description                                                                                              |
+| ------------------ | ----------------------- | -------------------------------------------------------------------------------------------------------- |
+| Created            | `"run:created"`         | When a run is created and enters the "Pending" state.                                                    |
+| Planning           | `"run:planning"`        | When a run acquires the lock and starts to execute.                                                      |
+| Needs Attention    | `"run:needs_attention"` | Human decision required. When a plan has changes and is not auto-applied, or requires a policy override. |
+| Applying           | `"run:applying"`        | When a run begins the apply stage, after a plan is confirmed or auto-applied.                            |
+| Completed          | `"run:completed"`       | When the run has completed on a happy path and can't go any further.                                     |
+| Errored            | `"run:errored"`         | When the run has terminated early due to error or cancellation.                                          |
+| Drifted            | `"assessment:drifted"`  | When an assessment of the workspace has found drift                                                      |
+| Drift Check Failed | `"assessment:failed`"   | When an assessment of the workspace has failed for some reason                                           |
 
 ## Notification Payload
 
+-> **Note:** Drift detection events do not yet support webhooks notifications
```

Comment:
I know they are coming, but we make it a habit of not doing future-facing statements in docs. :) Also I thought we weren't having a note since it's launching so soon? Should I add one to the docs as well?

```suggestion
-> **Note:** Drift detection events do not support webhooks notifications.
```